### PR TITLE
Feature/go back to previous element (ICC-38 & 39)

### PR DESCRIPTION
**Pull request by AugustGit**
## What's in this PR
Go back one element in Segment (Module or Strategy)
Updated Progress bar UI
Screen Shot 2021-01-06 at 2 29 07 PM
Screen Shot 2021-01-06 at 2 27 45 PM

## Prerequisites

- [ ] Documentation
- [ ] Tests added

- [ ] Jest tests passed (and nothing in red!)
- [ ] Detox tests passed
- [ ] PR & Commit Title and Message are formatted according to [Osedea Guidelines](https://github.com/Osedea/osedea_project_coding-standards/tree/master/git)
- [ ] Ready to Merge

## Design Changelog

Design | iOS | Android
--- | --- | ---
 |  |

### CODE REVIEW

File: src/pages/download/VideoDownloadPage.tsx [hunk 1]
```diff
@@ -18,7 +18,13 @@ const VideoDownloadPage: React.FC<IProps> = (props) => {
                 <Label>VideoDownloadPage Example</Label>
                 <OvalButton
                     text={'go to Login Page'}
-                    onPress={() => reset(STACKS.MAIN_STACK.LOGIN.LOG_IN_PAGE)}
+                    // onPress={() => reset(STACKS.MAIN_STACK.LOGIN.LOG_IN_PAGE)}
```
Comment (Helen2hang) [inline src/pages/download/VideoDownloadPage.tsx:21]: Change back to loginPage

File: src/pages/elements/EndPage.tsx [hunk 2]
```diff
@@ -44,8 +44,17 @@ const EndPage: React.FC<IProps> = (props) => {
...
-                    showBackButton={false}
+                    showBackButton={true}
                     showCloseButton={true}
+                    onBack={() =>
+                        reset(STACKS.MAIN_STACK.STACK, {
+                            screen:
+                                Navigation_Routes[section][segment][routeSwitch][
+                                    SegmentType.END_SEGMENT
+                                ].previous,
+                            params: {},
```
Comment (Helen2hang) [inline src/pages/elements/EndPage.tsx:55]: does it work without `params: {}`?

Comment (AugustGit) [reply src/pages/elements/EndPage.tsx:55]: We will 100% need the params to pass the routeSwitch for ex..the booster Session with ROUTES A,B,C

Comment (Helen2hang) [reply src/pages/elements/EndPage.tsx:55]: okie

**Review by Helen2hang — APPROVED** [review]

**Pull request merged**